neutone_sdk/gcn_1d.py

```diff
@@ -0,0 +1,381 @@
+import logging
+import os
+from typing import Optional
+
+import torch
+from torch import Tensor
+from torch import nn
+from torch.nn import functional as F
+
+logging.basicConfig()
+log = logging.getLogger(__name__)
+log.setLevel(level=os.environ.get("LOGLEVEL", "INFO"))
+
+
+class TFiLM(nn.Module):
+    def __init__(self, n_channels, cond_dim, tfilm_block_size, rnn_type="lstm"):
```

Comment:
Please add type hints to all method arguments for consistency in the SDK.

Reply:
Done, sorry about the oversight.

Reply:
Thanks, but I meant type hinting the parameters and return type for all methods, not just TFiLM, so there's quite a few still missing